examples/discovery-search-app/src/setupProxy.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const dotenv = require('dotenv');
+const serverEnv = dotenv.config({ path: path.join(__dirname, '../', '.server-env') });
+const clusterEnv = dotenv.config({ path: path.join(__dirname, '../', '.env.local') });
+const proxy = require('http-proxy-middleware');
+const { CloudPakForDataAuthenticator } = require('ibm-watson/auth');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+if (serverEnv.error) {
+  console.warn(serverEnv.error);
+}
+
+if (clusterEnv.error) {
+  console.warn(clusterEnv.error);
+}
+
+const RELEASE_PATH = process.env.RELEASE_PATH;
```

Comment:
we could probably just change this into 1 URL and then parse out the `BASE_URL` (since it should just be the protocol + host)